include/m_lib.h

```diff
@@ -52,6 +52,10 @@ typedef struct rgba_t { //can be put in other place
     u8 r, g, b, a;
 } rgba_t;
 
+typedef struct rgb_t {
```

Comment:
Not sure if this is the right spot for this, or if it should exists at all. Didn't find a place for the `u32` rgb color.